js/gem-notice.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
@fjarrett I think we do since .is-dismissible only adds the X and the animation and nothing else.